backends/traits/Cargo.toml

```diff
@@ -23,5 +23,5 @@ edition = "2018"
 
 [dependencies]
 zenoh = { version = "0.5.0-beta.5", path = "../../zenoh" }
-async-std = { version = "=1.8.0", features = ["unstable"] }
+async-std = { version = "=1.9.0", features = ["unstable"] }
```

Comment:
features = ["unstable"] was needed until async-std 1.8.0 to support the async channels. 
Since the channels have been stabilised in async-std 1.9.0, features = ["unstable"] should be hence removed.